drops/azure_edge_extensions_lnm_mq_isa95.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
+    "Summary": "This is a solution to build an ISA-95 network model on top of Azure IoT Operations utilizing AIO LNM and AIO MQ.",
+    "Description": "This repository defines two levels of an ISA-95 network model and enables MQTT bridging between the layers for bidirectional communication. This sample also showcases how to implement a Kafka Connector to retrieve data from a cloud-based Kafka Broker.",
+    "Cover": "",
+    "Authors": [
+        {
+            "Name": "Nebrass Lamouchi",
+            "Link": "https://github.com/nebrass"
+        },
+        {
+            "Name": "Marcel Bindseil",
+            "Link": "https://github.com/bindsi"
+        }
+    ],
+    "Source": "https://github.com/Azure-Samples/azure-edge-extensions-lnm-mq-isa95",
+    "Type": "sample_app",
+    "Difficulty": "Medium",
+    "ProgrammingLanguage": [
+        "YAML"
+    ],
+    "Products": [
+        "Azure IoT Operations",
+        "Arc",
+        "KafkaConnector"
+    ],
+    "LastModified": "2024-07-04T00:00:00.000Z",
+    "CreatedDate": "2024-07-04T00:00:00.000Z",
+    "Topics": []
```

Comment:
Maybe add Purdue model? ISA-95? Industrail newortking? thigs that can help the user to search your drop